Automattic/wc-calypso-bridge
# Replace appearance task with choosing theme

### Changes proposed in this Pull Request:

Partially addresses https://github.com/Automattic/wc-calypso-bridge/issues/1195.

This PR:

- Overhaul appearance task and append it after the products task
- Add appearance task fill, task header, and custom hook for action

Design:

![image](https://github.com/Automattic/wc-calypso-bridge/assets/3747241/df2d8b58-d9a4-4016-9be4-ff5dd49608db)



### How to test the changes in this Pull Request:

1. Complete products task
2. Go to WooCommerce > Home
3. Observe the appearance task matches design
4. Click on `Choose theme` or the appearance task itself
5. Observe you're navigated to `wordpress.com/themes/{ site_slug }`
6. Go back to WooCommerce > Home
7. Observe the appearance task is marked as complete

<img width="813" alt="image" src="https://github.com/Automattic/wc-calypso-bridge/assets/3747241/3dcd736f-7978-420c-b412-4dcb2839281a">




### Other information:




### FOR PR REVIEWER ONLY:
